source/includes/indexes/indexes.kt

```diff
@@ -51,4 +52,21 @@ fun main() {
         println(result)
     }
     // end-index-single-query
+
+    // start-index-compound
+    collection.createIndex(Indexes.ascending(Movie::type.name, Movie::genres.name))
+    // end-index-compound
+
+    // start-index-compound-query
+    val filter = and(
+        eq(Movie::type.name, "movie"),
+        `in`(Movie::genres.name, "Drama")
```

Comment:
This is a Kotlin (and probably Java) quirk – `in` is a language keyword AND a `Filters` method. To use the method as such you need to surround it with backticks.